src/components/images/draw.js

```diff
@@ -99,11 +99,11 @@ module.exports = function draw(gd) {
                 var dataURL = canvas.toDataURL('image/png');
 
                 thisImage.attr('xlink:href', dataURL);
+                resolve();
```

Comment:
Looks good!

Could you add a comment above to make sure we remember why that `resolve` is there? Something like:

```js
// resolve promise in onload handler instead of on 'load' to support IE11
// see https://github.com/plotly/plotly.js/issues/1685 
// for more details
```